plugins/core/core/src/software/aws/toolkits/core/utils/TextUtils.kt

```diff
@@ -17,6 +17,25 @@ fun convertMarkdownToHTML(markdown: String): String {
     return htmlRenderer.render(document)
 }
 
+fun extractCodeBlockLanguage(message: String): String {
+    // This fulfills both the cases of unit test generation(java, python) and general use case(Non java and Non python) languages.
+    val defaultTestGenResponseLanguage: String = "plaintext"
+    val indexStart: Int = 3
+    val codeBlockStart = message.indexOf("```")
```

Comment:
are there any other edge cases with this we need to worry about?

Reply:
- If we use this `extractCodeBlokLanguage`, we assume code generated should start or at-least will have 3 backticks with language name.
- So If generated code doesn't provide language after ```  we assume the language to be `plaintext`
- Even for counting number of code blocks, we are implementing this approach for recognizing code blocks with 3 backticks, until backend provide IDE this data.
- Right now this is used only for telemetry purposes